### PR TITLE
Add hands-on tutorial guide in Japanese

### PR DESCRIPTION
**Pull request by nakabonne**
**What this PR does / why we need it**:
This PR adds a hands-on step-by-step practical tutorial guide on `/ja/docs/hands-on/deploy-on-kubernetes/`.

**Which issue(s) this PR fixes**:

Fixes https://github.com/pipe-cd/pipe/issues/1810

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/ja/docs/hands-on/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+- 秘密鍵は後ほど利用するので控えておいてください
+
+### クラスタへPipedをインストールする
+お使いのクラスタ上にPipedをインストールします。前のステップで取得した以下のものが手元にあることを確認してください：
+
+- piped-id
+- piped-key
+- ssh-key
+
+[インストールガイド](/docs/operator-manual/piped/installation/#in-the-cluster-wide-mode)に従ってインストールを完了させてください。
```
Comment (nakabonne) [inline docs/content/ja/docs/hands-on/kubernetes.md:60]: I settled on just linking to the original guide for core steps.

**Comment by nakabonne** [comment]
Sorry, changed the page title and the link.

File: docs/content/ja/docs/hands-on/_index.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: "ハンズオン"
+linkTitle: "ハンズオン"
```
Comment (nghialv) [inline docs/content/ja/docs/hands-on/_index.md:3]: I wonder which is better: hands-on and tutorials and where to put this.

File: docs/content/ja/docs/hands-on/deploy-on-kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+---
+title: "Kubernetesへのデプロイ"
+linkTitle: "Kubernetesへのデプロイ"
```
Comment (nghialv) [inline docs/content/ja/docs/hands-on/deploy-on-kubernetes.md:3]: Kubernetesへのデプロイ　or  Kubernetes Applicationをデプロイ or Kubernetesへpipedをインストール
Which is better?

File: docs/content/ja/docs/hands-on/_index.md (hunk 2 above)
Comment (nakabonne) [reply docs/content/ja/docs/hands-on/_index.md:3]: It means that "ハンズオンチュートリアル" is better, right? If so one vote for that!

File: docs/content/ja/docs/hands-on/deploy-on-kubernetes.md (hunk 3 above)
Comment (nakabonne) [reply docs/content/ja/docs/hands-on/deploy-on-kubernetes.md:3]: I feel you. I understand the desire to clearly distinguish what to deploy to Kubernetes. The main reader of this guide can be considered as extremely newbies, who don't know exactly what Piped is. So how about "Kubernetesへ最初のアプリケーションをデプロイする"?

Comment (khanhtc1202) [reply docs/content/ja/docs/hands-on/deploy-on-kubernetes.md:3]: I think I'm okay with "Kubernetesへ最初のアプリケーションをデプロイする" 👍

**Comment by nghialv** [comment]
Let's merge this for now.
/approve

**Pull request merged**